src/docs/zksync-era.mdx

```diff
@@ -230,5 +227,8 @@ links:
     ###### Other
 
     - [hardhat-zksync-toolbox](https://era.zksync.io/docs/tools/hardhat/plugins.html#hardhat-zksync-toolbox) is a hardhat plugin that bundles all zkSync related Hardhat plugins
+    - [Remix plugin](https://medium.com/nethermind-eth/the-zksync-era-remix-plugin-a-how-to-guide-fc54e8d24bd3) for building and deploying contracts from your browser
+    - [web3.js zkSync plugin](https://github.com/web3/web3-plugin-zksync) extends web3.js with zkSync-specific RPC methods
```

Comment:
The following 2 tools are indeed tools related to zk sync however one important thing that we differentiate when we specify the tooling is whether the tooling was developed by the rollup team itself or it was developed by a 3rd party company. If we list tools that are developed by 3rd party companies we will end up with huge lists of tooling and there will be big noise in this section. If you go over the other rollups you will see that we list only tooling that is developed, maintained and released by the rollup team itself. 

That being said - if those plugins are developed by matter labs (which I do not think due to the organization where the repos are hosted) then we can add them in the list, otherwise we should remove them. 

The last entry for `foundry-zksync` is developed by matter labs so that's fine